examples/freely_decaying_mediterranean.jl

```diff
@@ -65,16 +65,19 @@ model = HydrostaticFreeSurfaceModel(; grid,
                               tracer_advection = WENO(grid; order = 7),
                                   free_surface = SplitExplicitFreeSurface(; cfl = 0.75, grid),
                                       buoyancy = SeawaterBuoyancy(),
-                                      tracers  = (:T, :S),
+                                      tracers  = (:T, :S, :c),
                                       coriolis = HydrostaticSphericalCoriolis(scheme = ActiveCellEnstrophyConserving()))
```

Comment:
```suggestion
                                      coriolis = HydrostaticSphericalCoriolis(scheme = ActiveCellEnstrophyConserving()))
```

Why isn't this default?

Reply:
Probably we should make it default